Vixen.sln

```diff
@@ -2305,8 +2305,8 @@ Global
 		{82D80094-F012-4248-ABA7-D55FB22807CD}.Debug|x64.Build.0 = Debug|x64
 		{82D80094-F012-4248-ABA7-D55FB22807CD}.Debug|x86.ActiveCfg = Debug|x86
 		{82D80094-F012-4248-ABA7-D55FB22807CD}.Debug|x86.Build.0 = Debug|x86
-		{82D80094-F012-4248-ABA7-D55FB22807CD}.Deploy|x64.ActiveCfg = Debug|x64
-		{82D80094-F012-4248-ABA7-D55FB22807CD}.Deploy|x64.Build.0 = Debug|x64
+		{82D80094-F012-4248-ABA7-D55FB22807CD}.Deploy|x64.ActiveCfg = Release|x64
+		{82D80094-F012-4248-ABA7-D55FB22807CD}.Deploy|x64.Build.0 = Release|x64
 		{82D80094-F012-4248-ABA7-D55FB22807CD}.Deploy|x86.ActiveCfg = Debug|x86

```

Comment:
I missed the debug ones on this one when I merged it also. Might as well fix it up as well.

Reply:
Edited file to specify release.